iam.tf

```diff
@@ -98,91 +97,7 @@ data "aws_iam_policy_document" "secure_env_vars" {
 }
 
 resource "aws_iam_policy" "secure_env_vars" {
-  count       = local.secure_env_vars_count
```

Comment:
this is always going to create the policy, even if `var.create_iam_role` is false. Do we want to have this lingering policy?

Reply:
I'm actually removing the mandatory secure env vars thing, so I'm adding the count back